torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -2112,7 +2112,8 @@ RegisterOperators reg2({
         [](Stack& stack) {
           auto index = pop(stack).toInt();
           auto string = pop(stack).toStringRef();
-          char c = string.at(index);
+          auto norm_index = normalizeIndex(index, string.size());
```

Comment:
can you modify the `__getitem__` overload for string instead? I think we plan to deprecate those prim operators and all using `aten::__getitem__` for indexing https://github.com/pytorch/pytorch/pull/22276